src/App.js

```diff
@@ -64,7 +66,7 @@ function App () {
       if (mounted) {
         fetchingHead()
       }
-    }, 5000)
+    }, 15000)
```

Comment:
why?

Reply:
it's causing to refresh more than necessary it seems  and polluting the logs a bit but it's fine - added it back